### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by gusta-glitch** [gusta-glitch/cmsp-bot#1] [General]
First release, all features are present in index of the repository.

**Full Changelog**: https://github.com/gusta-glitch/cmsp-bot/compare/0.2.1-alpha...1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/gusta-glitch/cmsp-bot/releases/tag/1.0.0'>v1.0.0</a>.</em>

**Comment by WinzLow95** [comment]
Eai Cara, eu gostei do seu Script, se você ver essa mensagem, pfvr entra em contato comigo para nós tentarmos conversar sobre, manda um email para d0smanlinux@gmail.com pfvr
